Sources/NIO/PendingDatagramWritesManager.swift

```diff
@@ -99,13 +102,19 @@ private func doPendingDatagramWriteVectorOperation(pending: PendingDatagramWrite
             storageRefs[c] = storageRef.retain()
             let addressLen = p.copySocketAddress(addresses.baseAddress! + c)
             iovecs[c] = iovec(iov_base: UnsafeMutableRawPointer(mutating: ptr.baseAddress!), iov_len: numericCast(toWriteForThisBuffer))
+            
+            let controlBytesRaw = UnsafeMutableRawBufferPointer(
+                rebasing: controlMessageBuffers[(c * controlBytesPerMessage)..<((c+1) * controlBytesPerMessage)])
```

Comment:
Can we write ourselves a helper for this slicing operation? It'd be helpful for avoiding repetition.